lib/api/calendars.ts

```diff
@@ -42,3 +42,11 @@ export const getInbox = (client: DAVClient) => async (userId: string): Promise<C
     ),
   }));
 }
+
+export const modifyEvent = (client: DAVClient) => async (eventPath: string, event: CalendarEventObject): Promise<string> => {
+  return await client.requestText({
+    url: `${BASE_PATH}/${eventPath}`,
```

Comment:
You should use `url-join`. Btw, I forgot to do the same for the `getInbox` method. Could you please do it for me 😛.